pkg/apis/core/v1alpha1/projectroletemplate_types.go

```diff
@@ -0,0 +1,218 @@
+/*
+Copyright 2020 The Bulward Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	rbacv1 "k8s.io/api/rbac/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// ProjectRoleTemplateSpec describes the desired state of ProjectRoleTemplate.
+type ProjectRoleTemplateSpec struct {
+	// Metadata	contains additional human readable ProjectRoleTemplate details.
+	Metadata *ProjectRoleTemplateMetadata `json:"metadata,omitempty"`
+	// BindTo defines the member types of the Project that this ProjectRoleTemplate will be bound to.
+	BindTo []BindingType `json:"bindTo,omitempty"`
+	// ProjectSelector selects Project objects that this ProjectRoleTemplate should be applied to.
```

Comment:
```suggestion
	// ProjectSelector selects applicable target projects
```

maybe a bit simpler?

Reply:
nice